netbox/templates/extras/object_imageattachments.html

```diff
@@ -0,0 +1,35 @@
+{% extends base_template %}
+{% load i18n %}
+{% load render_table from django_tables2 %}
+{% load thumbnail %}
+
+{% block extra_controls %}
+  {% if perms.extras.add_imageattachment %}
+    <a href="{% url 'extras:imageattachment_add' %}?object_type={{ object|content_type_id }}&object_id={{ object.pk }}" class="btn btn-primary">
+      <span class="mdi mdi-plus-thick" aria-hidden="true"></span> {% trans "Attach an Image" %}
+    </a>
+  {% endif %}
+{% endblock %}
+
+{% block content %}
+  <div class="d-flex flex-wrap">
+    {% for object in image_attachments %}
+      <div class="thumbnail m-2">
+        {% thumbnail object.image "200x200" crop="center" as tn %}
+          <a href="{{ object.get_absolute_url }}" class="d-block" title="{{ object.description }}">
+            <img
+              src="{{ tn.url }}"
+              width="{{ tn.width }}"
+              height="{{ tn.height }}"
+              class="rounded"
+              alt="{{ object.name }}"
+            />
+          </a>
+        {% endthumbnail %}
+        <div class="text-center text-secondary text-truncate fs-5">
+          {{ object }}
+        </div>
+      </div>
+    {% endfor %}
```

Comment:
Perhaps we should have an `{% empty %}` clause here noting that there are no image attachments.

I was a little surprised when I uploaded a front image for a device type (derp) and then viewed this tab and saw nothing. Admittedly, front/back images for device types are not the intended use for this tab.